### ISSUE TITLE
No response from host

### ISSUE THREAD

**Issue by vanFeidel** [nccgroup/umap#6]
I try this command:
./umap.py -P /dev/ttyUSB0 -i 
and all others (-O - have all the same answer. the goodfet.monitor test finish without a problem.

answer for ./umap.py -P /dev/ttyUSB0 -i 
## For help type: umap.py -h

01:01:00 - Audio : Audio control : PR Protocol undefined

**\* No response from host - check if the host is still functioning correctly ***

can some help me to fix this?!

**Comment by Rolalalalala** [comment]
Hi vanFeidel !
Same result for me, did u solve it ?
thks

**Comment by vanFeidel** [comment]
No, the problem was the hw
Am 10.05.2016 17:50 schrieb "Rolalalalala" notifications@github.com:

**Comment by Rolalalalala** [comment]
Thks for the answer. For me, I first tested on a linux target when it crashed, and after reinstalling and testing on windows target it worked well for me. U sure for the hw ?

**Comment by vanFeidel** [comment]
Yes, I have tested the hw very well and ist broken. I don't have any
responds from any host. Have a nice time with it ;) I'm glad that it works
Am 14.05.2016 11:01 PM schrieb "Rolalalalala" notifications@github.com:

**Comment by pqcheng** [comment]
>Thks for the answer. For me, I first tested on a linux target when it crashed, and after reinstalling and testing on windows target it worked well for me. U sure for the hw ?

Hey Rolalalala, which version of windows did you get it working on? I'm trying it on 10 and it's not working. There was a reddit post about pyserial not working on windows 10. http://forum.4dsystems.com.au/forum/forum-aa/4d-systems-hardware/4d-intelligent-display-modules-diablo16/50183-pyserial-working-in-win-7-not-win-10
I tried it on 8.1 earlier last week and it wasn't working either.

**Comment by PhenOMene** [comment]
Hi, 

> I try this command: ./umap.py -P /dev/ttyUSB0 -i and all others (-O - have all the same answer. the goodfet.monitor test finish without a problem.
> 
> answer for ./umap.py -P /dev/ttyUSB0 -i
> 
> ## For help type: umap.py -h
> 01:01:00 - Audio : Audio control : PR Protocol undefined
> 
> *** No response from host - check if the host is still functioning correctly ***
> 
> can some help me to fix this?!

Hi, 2023 and still the same issue ? any tips ? I m connected to a Linux

**Comment by BinyaminSharet** [comment]
> Hi,
> 
> > I try this command: ./umap.py -P /dev/ttyUSB0 -i and all others (-O - have all the same answer. the goodfet.monitor test finish without a problem.
> > answer for ./umap.py -P /dev/ttyUSB0 -i
> > ## For help type: umap.py -h
> > 01:01:00 - Audio : Audio control : PR Protocol undefined
> > *** No response from host - check if the host is still functioning correctly ***
> > can some help me to fix this?!
> 
> Hi, 2023 and still the same issue ? any tips ? I m connected to a Linux

Hi,

Sorry for the delayed response, the project was not active in the last few years, and I don't have the equipment to debug/maintain it anymore.
You're welcome to submit a pull request and I'll review and merge it
